server/jwt.go

```diff
@@ -31,7 +31,7 @@ var nscDecoratedRe = regexp.MustCompile(`\s*(?:(?:[-]{3,}[^\n]*[-]{3,}\n)(.+)(?:
 const jwtPrefix = "eyJ"
 
 // ReadOperatorJWT will read a jwt file for an operator claim. This can be a decorated file.
-func ReadOperatorJWT(jwtfile string) (*jwt.OperatorClaims, error) {
+func ReadOperatorJWT(jwtfile string) (*MergedClaimAndJWT, error) {
```

Comment:
You can't do that. This function is unfortunately public, so you would possibly break users that embed NATS Server and use this API.
If you need a function to return both the OperatorClaims and the JWT from reading the JWT file, and you think only the server right now would need it, create a private one that return both and have this function call it, but return only the OperatorClaims from it.

Reply:
I went this way as I had smth in the back of my mind that embedding the server and operator mode causes an error. 
But that was probably the wrong assumption. I fixed it